### PR TITLE
Fix ShellCheck errors

### PR DESCRIPTION
**Pull request by azzamsa**
# Pull Request

## Related issue
Fixes #687

## What does this PR do?
- ...

## PR checklist
Please check if your PR fulfills the following requirements:
- [ ] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [ ] Have you read the contributing guidelines?
- [ ] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: benchmarks/scripts/compare.sh [hunk 1]
```diff
@@ -19,20 +18,19 @@ fi
...
-        curl $file_s3_url --output $file_local_path --silent
-        if [[ "$?" -ne 0 ]]; then
+        if ! curl "$file_s3_url" --output "$file_local_path" --silent; then
             echo 'curl command failed.'
             exit 1
         fi
     fi
 done
 
-path_list=$(echo " $@" | sed 's/ / \/tmp\//g')
```
Comment (curquiza) [inline benchmarks/scripts/compare.sh:36]: Is it what we want @irevoire?

**Review by curquiza — COMMENTED** [review]
Thanks for the PR, i'm ok with it, I just have a doubt here 👇

**Review by irevoire — CHANGES_REQUESTED** [review]
Hey, I don’t really know which part did that. But the compare script doesn’t work with your PR

**Comment by curquiza** [comment]
Closing this due to lack of answer

**Pull request closed without merging**